utils/crypto.js

```diff
@@ -198,9 +199,30 @@ export const getMaxFee = routes => {
     return null
   }
   const fee = routes.reduce((max, b) => Math.max(max, b.total_fees), routes[0].total_fees)
+  return Math.ceil(Math.max(fee, 1))
+}
+
+/**
+ * getMaxFee - Given a list of routes, find the maximum fee factoring in all possible payment retry attempts.
+ *
+ * @param {*} routes List of routes
+ * @returns {number} maximum fee
+ */
+export const getMaxFeeInclusive = routes => {
+  let fee = getMaxFee(routes)
+
+  if (!routes || !routes.length) {
+    return null
+  }
 
-  // Add one to the fee to add room for accuracy error when using as a fee limit.
-  return fee + 1
+  // Factor in expontnial increment that could be applied in retry attempts.
+  const {
+    invoices: { retryCount, feeIncrementExponent },
+  } = config
+  range(retryCount).forEach(() => {
```

Comment:
fee*Math.pow(feeIncrementExponent,retryCount)

Reply:
This actually gives a slightly different result to what I wanted to achieve because we want to apply `Math.ceil` at each retry attempt.